### ISSUE TITLE
[Bug] 使用 Zeabur 模板部署的数据库版本如何更新呢？

### ISSUE THREAD

**Issue by Noah-Wu66** [lobehub/lobe-chat#5087]
### 📦 部署环境

Zeabur

### 📌 软件版本

LobeChat Database Version

### 💻 系统环境

Windows

### 🌐 浏览器

Chrome

### 🐛 问题描述

我是用 Zeabur 的模板部署了数据库版本[https://zeabur.com/templates/RRSPSD](url)，没有找到更新的方法，需要如何更新呢？

**Comment by SpeedupMaster** [comment]
[启动自动更新](https://lobehub.com/zh/docs/self-hosting/advanced/upstream-sync)

**Comment by Noah-Wu66** [comment]
不是 Fork 的仓库然后在 Zeabur 上部署的，而是使用 Zeabur 里面的模板功能，部署的是数据库版本，所以和仓库没有关系。

**Comment by xiao0yy** [comment]
![Image](https://github.com/user-attachments/assets/5e88da71-cdd8-45d2-9ec9-786a13f8c0c7)
